engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -468,14 +469,14 @@ ConstraintDeclarationException getMultipleGroupConversionsForSameSourceException
 
 	@Message(id = 133, value = "%1$s does not contain a constructor with the parameter types %2$s.")
 	ValidationException getBeanDoesNotContainConstructorException(@FormatWith(ClassObjectFormatter.class) Class<?> beanClass,
-			@FormatWith(CollectionOfClassesObjectFormatter.class) List<Class<?>> parameterTypes);
+			@FormatWith(ArrayOfClassesObjectFormatter.class) Class<?>[] parameterTypes);
```

Comment:
I don't see any value in this change. I saw at least once occurrence where you now convert the list to an array.

A list is fine, let's get rid of the diff noise.

Reply:
OK, sounds good then.